### ISSUE TITLE
Rails 3 Branch - Bundle install coughs errors on gemspec but still seems to work

### ISSUE THREAD

**Issue by jwoodlee** [NUARIG/surveyor#108]
Using surveyor (0.16.1) from git://github.com/breakpointer/surveyor.git (at rails3) 
surveyor at /Library/Ruby/Gems/1.8/bundler/gems/surveyor-60aa59db3037 did not have a valid gemspec.
This prevents bundler from installing bins or native extensions, but that may not affect its functionality.
The validation message from Rubygems was:
  ["spec/views/surveyor/show.html.haml_spec.rb"] are not files

Your bundle is complete! Use `bundle show [gemname]` to see where a bundled gem is installed.

**Comment by astjohn** [comment]
same here using the following in gemfile:

```
gem "surveyor", :git => "git://github.com/breakpointer/surveyor.git", :branch => "rails3"
```

**Comment by yoon** [comment]
vaid rails3 branch gemspec. closed by 20ebb9710766fb5c1d753508283c504f354a9779

**Comment by yoon** [comment]
Closed by: 20ebb9710766fb5c1d753508283c504f354a9779
